packages/component-test-utils-react/src/methods/dispatchEvent.js

```diff
@@ -0,0 +1,3 @@
+exports.dispatchEvent = (reactTree, eventName) => {
+  reactTree.props[`on${eventName}`]();
```

Comment:
Probably missing a mock of the `event` nop ?

Reply:
I'm dubtful. 

The event is what you should test.

Reply:
Do you have any use case ?

Reply:
Imagine:

```javascript
const Component = () => {
  const [inputValue, setInputValue] = useState('');
  const handleChange = useCallback(e => setInputValue(e.target.value)

  return <input type="text" onChange={handleChange} value={inputValue}/>
};
```